From: David Sterba <dsterba@suse.cz>
Date: Wed, 20 Mar 2013 13:31:27 +0000
Subject: [PATCH] btrfs: make orphan cleanup less verbose
Git-commit: 4884b476d769377c02f74fa83593e6afb562ad40
Patch-mainline: v3.10-rc2

The messages

  btrfs: unlinked 123 orphans
  btrfs: truncated 456 orphans

are not useful to regular users and raise questions whether there are
problems with the filesystem.

Signed-off-by: David Sterba <dsterba@suse.cz>
Signed-off-by: Josef Bacik <jbacik@fusionio.com>
---
 fs/btrfs/inode.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/fs/btrfs/inode.c
+++ b/fs/btrfs/inode.c
@@ -3235,9 +3235,9 @@ int btrfs_orphan_cleanup(struct btrfs_ro
 	}
 
 	if (nr_unlink)
-		btrfs_info(root->fs_info, "unlinked %d orphans", nr_unlink);
+		pr_debug("btrfs: unlinked %d orphans\n", nr_unlink);
 	if (nr_truncate)
-		btrfs_info(root->fs_info, "truncated %d orphans", nr_truncate);
+		pr_debug("btrfs: truncated %d orphans\n", nr_truncate);
 
 out:
 	if (ret)
